telegram/ext/filters.py

```diff
@@ -324,6 +324,65 @@ def filter(self, message):
 
     group = _Group()
 
+    class user(BaseFilter):
```

Comment:
after discussing about it with @bomjacob I remove my objection here.
I don't think it is "pretty" but it is reasonable and makes a clear API.